app/web/src/store/admin.store.ts

```diff
@@ -56,10 +52,11 @@ export const useAdminStore = () => {
             url: `${API_PREFIX}/workspaces/${workspaceId}/change_sets`,
           });
         },
+        // TODO(nick): remove in favor of the auth store call.
         async LIST_WORKSPACE_USERS(workspaceId: string) {
-          return new ApiRequest<{ users: AdminUser[] }>({
+          return new ApiRequest<{ users: WorkspaceUser[] }>({
             method: "get",
-            url: `${API_PREFIX}/workspaces/${workspaceId}/users`,
+            url: WORKSPACE_API_PREFIX.concat([workspaceId, "users"]),
```

Comment:
Removing this would introduce the auth store to the admin dashboard, which we should avoid until we know for sure where this route will land for the longer-term future.

Reply:
Endpoints can live in multiple stores without a problem...